aws/hll/aws-s3-transfer-manager/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "aws-s3-transfer-manager"
+version = "0.1.0"
+edition = "2021"
+authors = ["AWS Rust SDK Team <aws-sdk-rust@amazon.com>", "Aaron Todd <todaaron@amazon.com>"]
+description = "S3 Transfer Manager"
+license = "Apache-2.0"
+repository = "https://github.com/smithy-lang/smithy-rs"
+publish = false
+
+
+[dependencies]
+aws-sdk-s3 = { version = "1.22.0", features = ["behavior-version-latest", "test-util"] }
+aws-smithy-http = "0.60.7"
+tracing = "0.1"
+async-trait = "0.1.80"
+# FIXME - upgrade to hyper 1.x
+hyper = { version = "0.14.28", features = ["client"] }
```

Comment:
It's possible to use hyper 1.0 right now. We should do so if possible.

Reply:
We will. Baby steps. I figure we'll get it working and then start optimizing and checking off fixmes/todos as it makes sense.